experimental/Schemes/SpecOpen.jl

```diff
@@ -753,23 +733,23 @@ function SpecOpenMor(X::SpecType, d::RET, Y::SpecType, e::RET, f::Vector{RET}; c
   return SpecOpenMor(U, V, [SpecMor(U[1], Y, OO(U[1]).(f), check=check)], check=check)
 end
 
-function pullback(f::SpecOpenMor)
+function pullback(f::SpecOpenMor; check::Bool=true)
```

Comment:
Why introduce a `check` parameter here? This is a getter for an object associated to an already existing `SpecOpenMor`. That means: The `SpecOpenMor` has already been created and if anything, the checks should have been made then.

Reply:
It was for debugging, but didn't help.

Reply:
Yes, after I took up on it, I already guessed that this was the purpose. But that issue is resolved now and we should remove it.